### DISCUSSION TITLE
Missing tracing info in some traces

### DISCUSSION THREAD

**Discussion by david-almeida-capchase** [prisma/prisma#18951] [Q&A]
Hi!

I've come across a strange issue. I'm getting erratic behaviour with the tracing feature in prisma. 
I always see the prisma:client:operation, prisma:client:connect and prisma:client:serialize spans, but I randomly don't see in prisma:engine:connection, and most importantly, prisma:engine:db_query in some others.

This is pretty consistent, so the same queries give the same issues. I'm using nestjs, and the prisma client is used in multiple services as a singleton.

I've tested initialising a new prisma client for specific queries and then it shows up

**Comment by ludralph** [comment]
Hi @david-almeida-capchase 👋 

Can you please open a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=bug_report.md&title=) detailing this issue?

**Reply by david-almeida-capchase** [reply]
will do 👍
